hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/AbstractMapDataStore.java

```diff
@@ -16,6 +16,7 @@
 
 package com.hazelcast.map.impl.mapstore;
 
+import com.hazelcast.internal.nearcache.impl.invalidation.ToHeapDataConverter;
```

Comment:
Importing a nearcache internal class to mapstore package... That looks odd.

Reply:
fixed